test/testdrive/source-linear-operators.td

```diff
@@ -183,7 +183,7 @@ Explained Query:
         ArrangeBy keys=[[#0]]
           Project (#0, #2)
             Get l0
-  With
+  Where
```

Comment:
This is saying that we are removing `With` and adding `Where`, or am I misunderstanding?

Reply:
Sorry, too many late shifts recently, I fixed two `*.td` files that were actually correct.